src/main/scala/is/hail/expr/AST.scala

```diff
@@ -426,21 +426,44 @@ case class TupleConstructor(posn: Position, elements: Array[AST]) extends BaseSt
   } yield ir.MakeTuple(irElements)
 }
 
-case class ReferenceGenomeDependentConstructor(posn: Position, fName: String, grName: String, args: Array[AST]) extends AST(posn, args) {
+case class ReferenceGenomeDependentFunctions(posn: Position, fName: String, grName: String, args: Array[AST]) extends AST(posn, args) {
```

Comment:
This should be singular as its an invocation to a particular genome dependent function.